ee/tables/tufinfo/release_version_test.go

```diff
@@ -20,8 +19,6 @@ import (
 func TestTufReleaseVersionTable(t *testing.T) {
 	t.Parallel()
 
-	rand.Seed(time.Now().UnixNano())
```

Comment:
> rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: As of Go 1.20 there is no reason to call Seed with a random value. Programs that call Seed with a known value to get a specific sequence of results should use New(NewSource(seed)) to obtain a local random generator.